ThirdPartyAdapters/vungle/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.7.1.0
+- Verified compatibility with Vungle SDK 6.7.1.
+- Various Bug Fixes
```

Comment:
@jiancchen Can we add here following
"6.7.1 SDK has Support for Android 11 , and has some issues resolved in order to be used with API version 30 if required."

Reply:
@palrahul admob will fail though is any users set targetVersion to 30 atm, so i think we can add that to our IE doc also until admob is ready